core/src/consensus.rs

```diff
@@ -518,7 +529,11 @@ impl Tower {
     }
 
     pub fn refresh_last_vote_tx_blockhash(&mut self, new_vote_tx_blockhash: Hash) {
-        self.last_vote_tx_blockhash = Some(new_vote_tx_blockhash);
+        self.last_vote_tx_blockhash = BlockhashStatus::Blockhash(new_vote_tx_blockhash);
+    }
+
+    pub(crate) fn last_vote_tx_blockhash_failed(&mut self) {
```

Comment:
nit, feels like this needs an extra verb: maybe  `mark_last_vote_tx_blockhash_failed`